src/inspirehep-search-js/templates/literature/results.html

```diff
@@ -18,22 +18,39 @@ <h4 class="custom-h">
                 </h4>
                 <div class="brief-record-details">
                   <div class="authors">
-                    <!-- {{ render_record_authors(record, is_brief=true, show_affiliations=false) }} -->
-                    <span id="record-date">
-                      {{ record.display.date }}
-                    </span>
+                  <span ng-bind-html='record.metadata.collaboration | collaboration : record.metadata.corporate_author'></span>
+                  <span ng-bind-html='record.metadata.authors | authors : record.metadata.collaboration : record.metadata.corporate_author : record.display.number_of_authors : record.metadata.control_number'></span>
+                  <span id="record-date">
+                    - {{ record.display.date }}
+                  </span>
                   </div>
                   <div class="row">
-                    <div class="col-md-12 ">
-                      Journal info goes here
+                    <div ng-if="record.display.publication_info_line.pub_info &amp;&amp; record.metadata.dois"  class="col-md-12">
```

Comment:
It does not work directly with `&` ? In this [SO answer](http://stackoverflow.com/a/30855423/890185) they seem to use it .